lyricsgenius/genius.py

```diff
@@ -520,6 +521,10 @@ def find_artist_id(search_term):
             return None
 
         artist_info = self.artist(artist_id)['artist']
+        found_name = artist_info['name']
+        if found_name != artist_name and allow_name_change:
```

Comment:
Sorry for being unclear. As `allow_name_change` isn't used anymore, it's best to also remove it from this if statement.

Reply:
I'm confused: why should the print statement be kept if its behaviour is going to change?

Reply:
Just to inform the user in case the artist's name is changed.
```python
if found_name != artist_name and self.verbose:
    print("Changing artist name to '{a}'".format(a=safe_unicode(found_name)))
```